gef.py

```diff
@@ -10055,28 +9847,28 @@ def do_invoke(self, argv):
             self.usage()
             return
 
-        info("{:s} for '{:s}'".format(self._cmdline_, filename))
+        info(f"{self._cmdline_} for '{filename}'")
         self.print_security_properties(filename)
         return
 
-    def print_security_properties(self, filename):
+    def print_security_properties(self, filename: str) -> None:
         sec = checksec(filename)
         for prop in sec:
             if prop in ("Partial RelRO", "Full RelRO"): continue
             val = sec[prop]
             msg = Color.greenify(Color.boldify(TICK)) if val is True else Color.redify(Color.boldify(CROSS))
             if val and prop == "Canary" and is_alive():
                 canary = gef_read_canary()[0]
```

Comment:
deprecated and also canary can be `None`. Dirty quick fix would be to use a default value of 0 or something:

```suggestion
                canary = gef.session.canary[0] if gef.session.canary else 0
```